package/kernel/bcm63xx-cferam/Makefile

```diff
@@ -0,0 +1,42 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=bcm63xx-cferam
+PKG_VERSION:=2020-05-10
```

Comment:
Is this really a version or might it be more appropriate to use PKG_SOURCE_DATE?

Reply:
You're right, I'll change it